core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitState.java

```diff
@@ -33,5 +33,11 @@ public enum CircuitState {
      * If it doesn't complete within a certain time, another trial request will be sent again.
      * All other requests fails immediately same as OPEN.
      */
-    HALF_OPEN
+    HALF_OPEN,
```

Comment:
Where is FORCE_CLOSED state?

Reply:
I was wondering why you needed `FORCE_CLOSED`.
There is a situation where a request should be sent to a target even if the target is unavailable?

Reply:
Well edgecase of a severely misconfigured CB which keeps opening. Perhaps better than "FORCE_CLOSED" would be "DISABLED" - instead of force closing it, pretty much disable the CB and make it pass through

Reply:
Make sense. Would you mind opening an issue for that?